src/commands/yarn_build_image.yml

```diff
@@ -0,0 +1,20 @@
+description: >
+  Run `yarn build`, build docker image and store it in workspace.
+
+parameters:
+  repository_name:
+    type: string
+    description: "Docker repository name"
+  repository_url:
+    type: string
+    description: "Docker repository URL"
+    default: "637192944017.dkr.ecr.us-east-1.amazonaws.com"
+
+steps:
+  - yarn_build
+  - run:
+      name: "Yarn install production dependencies"
+      command: yarn install --prod --pure-lockfile
```

Comment:
Ještě si říkám, že bych i tenhle yarn install mohl obalit restorem a uložením cache. 🤔

Reply:
Jinak, teda, imho správnější by bylo ty produkční package instalovat až v Dockerfilu, jenže se tam instalují `fgr-lib-backend` a další přes ssh z privátních git repozitářů, ale to bych musel nějak do dockeru dostat ssh klíč a to ani nevím jestli jde nějak rozumně vyřešit, tak aspoň takto. 🙂

Reply:
Tak vyřešily by to github packages 🙄